hermes-api/src/main/java/pl/allegro/tech/hermes/api/endpoints/OAuthProviderEndpoint.java

```diff
@@ -0,0 +1,39 @@
+package pl.allegro.tech.hermes.api.endpoints;
+
+import pl.allegro.tech.hermes.api.OAuthProvider;
+import pl.allegro.tech.hermes.api.PatchData;
+
+import javax.ws.rs.*;
```

Comment:
\[Checkstyle\] ERROR: Using the '\.\*' form of import should be avoided \- javax\.ws\.rs\.\*\.